src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -73,6 +85,23 @@ public void populatePanel(CellGrid grid, GameSession gameSession) {
             }
         }
     }
+    
+    public void remarkGivenCells(CellGrid grid){
```

Comment:
Hmmmm I'm not seeing that. If it doesn't work, there must be some other explanation. Can you debug it and find out?